terraform/files/attach-data-volume.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# adapted from
+# https://github.com/hashicorp/terraform/issues/2740#issuecomment-288549352
+
+set -e
+set -x
+
+# note the name doesn't match the device_name in Terraform
+DEVICE=/dev/xvdf
+OWNER=www-data
+DEST=/usr/share/wordpress/wp-content
+OLDDEST=$DEST-old
+
+devpath=$(readlink -f $DEVICE)
+
+if [[ $(sudo file -s $devpath) != *ext4* && -b $devpath ]]; then
+  # Filesystem has not been created. Create it!
+  sudo mkfs -t ext4 $devpath
+fi
+
+sudo mv $DEST $OLDDEST
+sudo mkdir -p $DEST
+
+echo "$devpath $DEST ext4 defaults,nofail,noatime,nodiratime,barrier=0,data=writeback 0 2" | sudo tee -a /etc/fstab > /dev/null
```

Comment:
Should probably check that it doesn't exist before appending to fstab, yeah?

Reply:
Will look into the best way to do that. This script got complex enough that I'm considering changing it to Ansible (as a follow-up), which takes care of checks like that.

Reply:
(but let me know if you know of an example)

Reply:
Should be able to do with an egrep and regex:
```
if ! egrep "^${devpath}" /etc/fstab; then
  echo "$devpath $DEST ... etc
fi
```